src/scss/components/button/_index.scss

```diff
@@ -1,3 +1,3 @@
-@import "mixins";
+@import "config";
```

Comment:
remove import, given that it doesn't generate CSS on its own and it's not being used explicitly in this file